test/test_rtfcre_dict.py

```diff
@@ -31,8 +31,8 @@ def test_converter(self):
         (r'\}', '}'),
         (r'\~', '{^ ^}'),
         (r'\_', '-'),
-        ('\\\r', '{#Return}{#Return}'),
-        ('\\\n', '{#Return}{#Return}'),
```

Comment:
If we did this, then we wouldn't be able to save escaped newlines from Plover without it becoming paragraph strokes on next load.